public/bitemp_graph.js

```diff
@@ -695,6 +730,59 @@ var barChart = function() {
       //top horizontal line
       lineCreator(0, width - margin.left, 3, 3, dragDown, 'dragDown');
       $('#endValBox').val(format(yScale.invert(0)));
+      
+      writeQuery();
+      createFilledRectangle();
+
+      function createFilledRectangle() {
+        $('#filledRect').remove();
+          x1 = $('#dragLeft')[0].__data__.x;
+          x2 = $('#dragRight')[0].__data__.x;
+          y1 = $('#dragUp')[0].__data__.y;
+          y2 = $('#dragDown')[0].__data__.y;
+
+          var x, y, widthRect, heightRect;
+          if(getSelected('valDropdown') !== 'None' && getSelected('sysDropdown') !== 'None') {
+            x = x2+7;
```

Comment:
7 is the width of a draggable bar. So we didn't want the draggable bar to be included in the highlighted rectangle